docs/get_started.rst

```diff
@@ -1,17 +1,345 @@
+
+    $ cat cumulusci.yml
+    project:
+        name: MyRepoName
+        package:
+            name: My Repo Name
+            namespace: mynamespace
+
+The newly created `cumulusci.yml` file is the configuration file for your project specific tasks, flows, and CumulusCI customizations.
+For more information regarding configuraiton, checkout our `project configuration <#TODO internal ref here>`_ section of the docs.
+You can add and commit it to your git repository:
+
+.. code-block:: console
+
+    $ git add cumulusci.yml
+    $ git commit -m "Initialized CumulusCI Configuration"
+
+
+
+Add Your Repo to GitHub
+^^^^^^^^^^^^^^^^^^^^^^^
+With your ``cumulusci.yml`` file committed, we now want to create a repository on GitHub for our new project and push our changes there.
+
+#. `Create a new repository <https://docs.github.com/en/free-pro-team@latest/articles/creating-a-new-repository>`_ on GitHub.
+#. At the top of your GitHub Repository's Quick Setup page, click the clipboard button to copy the remote repository URL.
+#. In your terminal, `add the URL for the remote repository <https://docs.github.com/en/free-pro-team@latest/articles/adding-a-remote>`_ to where your local repository will be pushed::
+
+    $ git remote add origin <remote_repository_url>
+
+#. Verify the remote was added successfullly with::
+
+    $ git remote -v
+
+#. `Push the changes <https://docs.github.com/en/free-pro-team@latest/github/using-git/pushing-commits-to-a-remote-repository>`_ in your local repository to GitHub::
+
+    $ git push -u origin master
+
+
+
+Convert an Existing Salesforce Project
+--------------------------------------
+
+Project Setup
+^^^^^^^^^^^^^
+#. Create a directory for your project to live in, and navigate to it::
+
+    $ mkdir mySalesforceProject; cd mySalesforceProject
+
+#. Initialize the directory as a git repository::
+
+    $ git init
+    Initialized empty Git repository in /Users/MrCCI/repos/mySalesforceProject/.git/
+
+#. Initialize the repository as a CumulusCI project. See `project initialization`_.
+
+
+Extracting Your Project's Metadata
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Next we need to extract your package metadata.
+Depending on what type of org your project is in, there are different methods for extracting the desired pieces of metadata.
+
+Production or Developer Edition Orgs
+*******************************************
+#. Create a package for your - If you're looking to deploy back to the same org, or would like to be able to use ``cci task run retrieve_source`` ensure that your project's name in ``cumulusci.yml`` matches the name of your package.
+#. Add the desired metadata components to the package.
+#. Extract package in either metadata (``src/``) or ``sfdx`` (``force-app/``) formats.
+    * For metadata format::
+
+        $ cci task run retrieve_source
+
+    * For ``sfdx`` format, replace <namespace> with the namespace of your package::
+
+        $ sfdx force:source:retrieve -n <namespace>
```

Comment:
This is tricky because if the persistent org is authenticated in CCI, it is not also authenticated in SFDX. Perhaps we should recommend an MDAPI retrieve followed by a convert?

Reply:
We should probably also add a brief discussion of creating scratch org definitions in an attempt to match the existing org.

Reply:
For your latter comment, do you think that item should be moved out of the "Other Considerations" section at the bottom?

Reply:
I think it probably fits best as a subsection under the Developer Org/Sandbox migration section (see above). What do you think?